tests/Fixer/ClassNotation/OrderedImplementsFixerTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\ClassNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Dave van der Brugge <dmvdbrugge@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\ClassNotation\OrderedImplementsFixer
+ */
+final class OrderedImplementsFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixAlphaCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixAlpha($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixAlphaCases()
+    {
+        return [
+            'single' => [
+                '<?php class T implements A {}',
+            ],
+            'multiple' => [
+                '<?php class T implements A, B, C {}',
+                '<?php class T implements C, A, B {}',
+            ],
+            'newlines' => [
+                "<?php class T implements\nB,\nC\n{}",
+                "<?php class T implements\nC,\nB\n{}",
+            ],
+            'newlines and comments' => [
+                "<?php class T implements\n// Here's A\nA,\n// Here's B\nB\n{}",
+                "<?php class T implements\n// Here's B\nB,\n// Here's A\nA\n{}",
+            ],
+            'no whitespace' => [
+                '<?php class T implements/*An*/AnInterface/*end*/,/*Second*/SecondInterface{}',
+                '<?php class T implements/*Second*/SecondInterface,/*An*/AnInterface/*end*/{}',
+            ],
+            'FQCN' => [
+                '<?php class T implements \F\Q\C\N, \F\Q\I\N {}',
+                '<?php class T implements \F\Q\I\N, \F\Q\C\N {}',
+            ],
+            'mixed' => [
+                '<?php class T implements \F\Q\C\N, Partially\Q\C\N, /* Who mixes these? */ UnNamespaced {}',
+                '<?php class T implements /* Who mixes these? */ UnNamespaced, \F\Q\C\N, Partially\Q\C\N {}',
+            ],
+            'nested anonymous classes' => [
```

Comment:
since we still support >PHP7 on master I'm afraid you have to move this to separate test and provide for PHP7 and up :(